db/migrate/20151230082305_add_container_event_ems_ref.rb

```diff
@@ -0,0 +1,5 @@
+class AddContainerEventEmsRef < ActiveRecord::Migration
+  def change
+    add_column :event_streams, :ems_ref, :string
```

Comment:
@simon3z It isn't needed in the table but some sort of key should be parsed in order to maintain consistency with the way VMs are processed and link ids to (```find_by_ems_ref_and_ems_id``` that requires ```ems_ref_key``` for event) . I'm removing the ems_ref column and also removing it from the event hash after finish using it.
I was trying to avoid making ```self.process_container_entities_in_event!```  too complex, if you have a better proposal I'm open to hear it,